vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -727,22 +727,91 @@ forms: a_form_name"
 )
 ```
 
+# Merging forms in the `secuTrialdata` object
+
+Naturally, you will sometimes need to merge/join some of the data from the individual form data stored
+in your `secuTrialdata` object. To achieve this you can use the `merge()` function.
```

Comment:
maybe specify that merge is from base?

Reply:
Haha, I actually had it there and decided to remove it. I can put it back in though.